functionary/ui/templates/partials/modal.html

```diff
@@ -0,0 +1,17 @@
+<div class="modal is-active" _="on closeModal remove me">
+    <div class="modal-background" _="on click trigger closeModal"></div>
+    <div class="modal-card">
+        <header class="modal-card-head">
+            <h1 class="modal-card-title is-2">{{ title }}</h1>
+            <button class="delete" aria-label="close" _="on click trigger closeModal"></button>
+        </header>
+        <section class="modal-card-body">
+            <div class="subtitle is-6">
+                {{ description }}
```

Comment:
This would be more templatey as {% block description %}{% endblock description %}

Reply:
Fixed